### PR TITLE
new test for LFC stats in explain

### PR DESCRIPTION
**Pull request by Bodobolero**
## Problem

PR https://github.com/neondatabase/neon/pull/6851 implemented new output in PostgreSQL explain.
this is a test case for the new function.

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] If it is a core feature, I have added thorough tests.
- [no ] Do we need to implement analytics? if so did you add the relevant metrics to the dashboard?
- [no] If this PR requires public announcement, mark it with /release-notes label and add several sentences in this section.

## Checklist before merging

- [ ] Do not forget to reformat commit message to not include the above checklist

### CODE REVIEW

**Review by knizhnik — APPROVED** [review]
I have not tested it, but test looks correct.

**Comment by Bodobolero** [comment]
> I have not tested it, but test looks correct.

I have tested it.

File: test_runner/regress/test_explain_with_lfc_stats.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+from fixtures.log_helper import log
+from fixtures.neon_fixtures import NeonEnv
+
+
+def test_explain_with_lfc_stats(neon_simple_env: NeonEnv):
+    env = neon_simple_env
+
+    cache_dir = os.path.join(env.repo_dir, "file_cache")
+    os.mkdir(cache_dir)
```
Comment (bayandin) [inline test_runner/regress/test_explain_with_lfc_stats.py:11]: Let's use `Path` here (it requires to add `from pathlib import Path`):

```suggestion
    cache_dir = env.repo_dir / "file_cache"
    cache_dir.mkdir()
```

**Review by bayandin — APPROVED** [review]
Looks good, just a couple of style comments

File: test_runner/regress/test_explain_with_lfc_stats.py (hunk 1 above)
Comment (Bodobolero) [reply test_runner/regress/test_explain_with_lfc_stats.py:11]: ok - actually I had copied that line from another test ;-)

File: test_runner/regress/test_explain_with_lfc_stats.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
+from pathlib import Path
+
+from fixtures.log_helper import log
+from fixtures.neon_fixtures import NeonEnv
+
+
+def test_explain_with_lfc_stats(neon_simple_env: NeonEnv):
+    env = neon_simple_env
+
+    cache_dir = Path(env.repo_dir) / "file_cache"
```
Comment (vadim2404) [inline test_runner/regress/test_explain_with_lfc_stats.py:10]: python has this built-in module https://docs.python.org/3/library/tempfile.html

Comment (vadim2404) [reply test_runner/regress/test_explain_with_lfc_stats.py:10]: oh, nevermind, you just created a folder, not the file itself

File: test_runner/regress/test_explain_with_lfc_stats.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        "insert into pgbench_accounts(aid,bid,abalance,filler) select aid, (aid - 1) / 100000 + 1, 0, '' from generate_series(1, 100000) as aid;"
+    )
+
+    log.info(f"warming up caches with sequential scan in {endpoint.connstr()}")
+    cur.execute("SELECT * FROM pgbench_accounts WHERE abalance > 0")
+
+    log.info("running explain analyze without LFC values to verify they do not show up in the plan")
+    cur.execute("EXPLAIN (ANALYZE, BUFFERS) SELECT * FROM pgbench_accounts WHERE abalance > 0")
+    rows = cur.fetchall()
+    plan = "\n".join([r[0] for r in rows])
```
Comment (vadim2404) [inline test_runner/regress/test_explain_with_lfc_stats.py:54]: NIT: 

```python
plan = "\n".join(r[0] for r in rows)
```

you don't need to build the list, you can pass the generator

Comment (Bodobolero) [reply test_runner/regress/test_explain_with_lfc_stats.py:54]: ok

**Review by vadim2404 — APPROVED** [review]

**Pull request merged**